crates/common_utils/src/types.rs

```diff
@@ -3,6 +3,8 @@ pub mod keymanager;
 
 /// Enum for Authentication Level
 pub mod authentication;
+/// Enum for Theme Lineage
+pub mod theme;
```

Comment:
I'd prefer to avoid having this in `common_utils` crate if possible, since this crate is either being pulled as a direct or a transitive dependency by a good number of our repositories. Can we move this to `common_enums` instead?

Reply:
Yeah, I'd also like to put in `common_enums` and tried this first but `id_type` is defined in `common_utils` and it is not available in `common_enums`.

I want this enum to be available in DB files and API files and only this is the common dependency where I can add something like this.

Let me know, if there is any other crate where I can place this.